docs/remote-caching.md

```diff
@@ -39,18 +39,18 @@ The key benefit of this approach is that you get the upload for free if you anyw
 
 #### How to use inline caching
 
-In order to enable inline caching, simply add `--ci` in your invocation of `earthly` in your CI, or `--use-inline-cache` on individual developer's machines. If the `--push` command is also specified, the use of the cache will be read-write.
+In order to enable inline caching, add `--ci --use-inline-cache --save-inline-cache` in your invocation of `earthly` in your CI, or `--use-inline-cache` on individual developer's machines. If the `--push` command is also specified, the use of the cache will be read-write.
```

Comment:
Perhaps this can be unified a bit, to no longer make the distinction between CI and non-CI?

Reply:
Yeah, makes sense. All of the inline caching stuff is distinct from `--ci` now, so maybe we don't need to reference the `--ci` flag at all.